src/firefly/js/ui/HiPSViewer.jsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
```

Comment:
HiPSViewer is not the right name of this file.  How about `HiPSSurveyListDisplay.jsx`.  HiPSViewer sounds like it is the viewer not the list of surveys.

Reply:
We do need to make the popular list settable by different applications, even the same application at different deployment sites. I can see SUIT portal at NCSA and Chile setting different sites due to network.